molecule/playbooks/prepare.yml

```diff
@@ -147,3 +157,11 @@
         state: present
         exclusive: False
       when: (ansible_distribution == "Ubuntu") and (lookup('env', 'DESTROY_ENV') == "no")
+
+    - name: Setup SSH keys Amazon
```

Comment:
Let's add AL as additional condition to ec2-user task instead of having one more task

Reply:
Fixed